app/views/shared/_post_errors.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if post.errors.any? %>
+  The form contains <%= pluralize(post.errors.count, "error") %>.
+  <ul>
+    <% post.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+  </ul>
+<% end %>
```

Comment:
Cool, yeah now that I have experience with doing that for the thing you mentioned above, that sounds good.